src/macropad/index.html

```diff
@@ -101,81 +97,129 @@ <h3>Tools</h3>
+        />
       </figure>
       <p>Solder the diodes, and trim the legs with flush cutters.</p>
       <figure>
-        <img src="trimmeddiodes.jpg" width="600" height="400" alt="Diodes soldered and trimmed"/>
+        <img
+          src="trimmeddiodes.jpg"
+          width="600"
+          height="400"
+          alt="Diodes soldered and trimmed"
+        />
       </figure>
       <h4>Switches & Pro Micro</h4>
       <p>Place and solder the lower four switches</p>
       <figure>
-        <img src="fourswitches.jpg" width="600" height="400" alt="Lower four switches soldered in place"/>
-      <figure>
+        <img
+          src="fourswitches.jpg"
+          width="600"
+          height="400"
+          alt="Lower four switches soldered in place"
+        />
+      </figure>
       <p>
-        Tack on the two pin headers, fiddle with them until they're straight
-        and the pro micro fits on.
+        Tack on the two pin headers, fiddle with them until they're straight and
+        the pro micro fits on.
       </p>
       <figure>
-        <img src="tackedheader.jpg" width="600" height="400" alt="Headers tacked into place"/>
+        <img
+          src="tackedheader.jpg"
+          width="600"
+          height="400"
+          alt="Headers tacked into place"
+        />
       </figure>
       <figure>
-        <img src="testfit.jpg" width="600" height="400" alt="Pro Micro aligned on the headers"/>
-      <figure>
+        <img
+          src="testfit.jpg"
+          width="600"
+          height="400"
+          alt="Pro Micro aligned on the headers"
+        />
+      </figure>
       <p>Solder the pin header.</p>
       <figure>
-        <img src="solderedheaders.jpg" width="600" height="400" alt="Headers soldered"/>
+        <img
+          src="solderedheaders.jpg"
+          width="600"
+          height="400"
+          alt="Headers soldered"
+        />
       </figure>
       <p>Solder the top two switches.</p>
       <figure>
-        <img src="twoswitches.jpg" width="600" height="400" alt="Top two switches soldered"/>
+        <img
+          src="twoswitches.jpg"
+          width="600"
+          height="400"
+          alt="Top two switches soldered"
+        />
       </figure>
       <p>Solder the Pro Micro onto the headers</p>
       <figure>
-        <img src="solderedpromicro.jpg" width="600" height="400" alt="Pro Micro soldered"/>
+        <img
+          src="solderedpromicro.jpg"
+          width="600"
+          height="400"
+          alt="Pro Micro soldered"
+        />
       </figure>
       <h4>Flash the Pro Micro</h4>
-      <pre>$ git clone git@github.com:qmk/qmk_firmware.git
+      <pre>
+$ git clone git@github.com:qmk/qmk_firmware.git
 
 $ cd qmk_firmware
 
 $ ./util/linux_install.sh
 
 $ sudo make pdxkbc:default:avrdude
 
-# short the RST and GND pins on the pro micro when prompted</pre>
+# short the RST and GND pins on the pro micro when prompted</pre
+      >
```

Comment:
```suggestion
# short the RST and GND pins on the pro micro when prompted
      </pre>
```

Reply:
This will add a newline to the end of the code snippet (since `<pre>` interprets all whitespace literally and displays it)

Reply:
well THAT'S DUMB.

Reply:
yeah i wish there was `<pre>` but it ignored leading and trailing whitespace